src/csharp/Microsoft.Spark/ML/Linalg/SparseVectorUDT.cs

```diff
@@ -0,0 +1,75 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Spark.Sql;
+using Microsoft.Spark.Sql.Types;
+using Newtonsoft.Json.Linq;
+
+namespace Microsoft.Spark.ML.Linalg
+{
+    /// <summary>
+    /// The data type for a SparseVectorUDT, used by SparseVector. 
+    /// </summary>
+    public class SparseVectorUDT : DataType
+    {
+        /// <summary>
+        /// Returns JSON object describing this type.
+        /// </summary>
+        internal override object JsonValue =>
+            new JObject(
+                new JProperty("type", "udt"),
+                new JProperty("class", "org.apache.spark.ml.linalg.VectorUDT"),
+                new JProperty("pyClass", "pyspark.ml.linalg.VectorUDT"),
+                new JProperty("sqlType", new JObject(
+                    new JProperty("type", "struct"),
+                    new JProperty("fields", new JArray(
```

Comment:
Will there be any issues not including the `"metadata": {}` property to each of these fields ?